### PR TITLE
NMS-13314: Prevent meta-data modification

### PR DESCRIPTION
**Pull request by fooker**
Modification of meta-data was only intended to be done via requisitions.
The requisitions process will create, update and delete meta-data of a node
according to the values defined in a requisition. Changing the meta-data
after a requisition synchronization was never intended conceptually. It
suffered from loosing of meta-data after re-synchronization. Keeping the
data would require manual deletion in massive scale using the UI and
leads to desync with the expected values.

This also reverts #3291.

### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-13314

### CODE REVIEW

File: docs/modules/development/pages/rest/meta-data.adoc [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [[metadata-rest]]
 = Metadata
 
-You can qurey, add, modify, or delete the actual metadata of nodes, interfaces, and services via REST endpoints.
+You can qurey the actual metadata of nodes, interfaces, and services via REST endpoints.
 
 [[rest-api-meta-data-get]]
 == GETs (Reading Meta-Data)
```
Comment (Bonrob2) [inline docs/modules/development/pages/rest/meta-data.adoc:7]:
```suggestion
== GETs (Reading Metadata)
```

**Review by Bonrob2 — CHANGES_REQUESTED** [review]
one minor change

File: docs/modules/development/pages/rest/meta-data.adoc [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [[metadata-rest]]
 = Metadata
 
-You can qurey, add, modify, or delete the actual metadata of nodes, interfaces, and services via REST endpoints.
+You can qurey the actual metadata of nodes, interfaces, and services via REST endpoints.
```
Comment (Bonrob2) [inline docs/modules/development/pages/rest/meta-data.adoc:4]:
```suggestion
You can query the actual metadata of nodes, interfaces, and services via REST endpoints.
```

**Review by Bonrob2 — CHANGES_REQUESTED** [review]
one more. :-)

**Review by Bonrob2 — APPROVED** [review]
LGTM!

**Comment by Naicisum** [comment]
Code wise, I think this should go into H29 as it's a breaking change and H28 has been released. This API has been in the wild for a while and it is unknown who would be impacted by this change.

**Comment by indigo423** [comment]
> Code wise, I think this should go into H29 as it's a breaking change and H28 has been released. This API has been in the wild for a while and it is unknown who would be impacted by this change.

IIRC we try to follow semver.org and with the following guidelines:

> Given a version number MAJOR.MINOR.PATCH, increment the:
  * MAJOR version when you make incompatible API changes,
  * MINOR version when you add functionality in a backward-compatible manner, and
  * PATCH version when you make backward-compatible bug fixes.

**Pull request merged**